tests/drone/config.json

```diff
@@ -3,8 +3,8 @@
   "theme": "owncloud",
   "version": "0.1.0",
   "openIdConnect": {
-    "metadata_url": "https://konnectd:9130/.well-known/openid-configuration",
-    "authority": "https://konnectd:9130",
+    "metadata_url": "https://ocis:9200/.well-known/openid-configuration",
+    "authority": "https://ocis:9200",
```

Comment:
Dont make changes in this file. We should leave this as it is for the tests with oc10. 
you need to change `tests/drone/ocis-config.json` file